docs/monitors/aspdotnet.md

```diff
@@ -33,37 +33,42 @@ Monitor Type: `aspdotnet`
 
 ## Configuration
 
+**For a list of monitor options that are common to all monitors, see [Common
+Configuration](../monitor-config.md#common-configuration).**
+
+
 | Config option | Required | Type | Description |
 | --- | --- | --- | --- |
 | `counterRefreshInterval` | no | `int64` | (Windows Only) Number of seconds that wildcards in counter paths should be expanded and how often to refresh counters from configuration. (**default:** `60s`) |
 | `printValid` | no | `bool` | (Windows Only) Print out the configurations that match available performance counters.  This is used for debugging. (**default:** `false`) |
 
 
-
-
 ## Metrics
 
-The following table lists the metrics available for this monitor. Metrics that are marked as Included are standard metrics and are monitored by default.
-
-| Name | Type | Included | Description |
-| ---  | ---  | ---    | ---         |
-| `asp_net.application_restarts` | gauge |  | Count of ASP.NET application restarts. |
-| `asp_net.applications_running` | gauge |  | Number of running ASP.NET applications. |
-| `asp_net.requests_current` | gauge |  | Current number of ASP.NET requests. |
-| `asp_net.requests_queue` | gauge |  | Number of queued ASP.NET requests. |
-| `asp_net.requests_rejected` | gauge |  | Count of rejected ASP.NET requests. |
-| `asp_net.worker_process_restarts` | gauge |  | Count of ASP.NET worker process restarts. |
-| `asp_net.worker_processes_running` | gauge |  | Number of running ASP.NET worker processes. |
-| `asp_net_applications.errors_during_execution` | gauge |  | Count of errors encountered by ASP.NET application durring execution. |
-| `asp_net_applications.errors_total_sec` | gauge |  | Error rate per second for the given ASP.NET application. |
-| `asp_net_applications.errors_unhandled_during_execution_sec` | gauge |  | Unhandled error rate per second countered while an ASP.NET application is running. |
-| `asp_net_applications.pipeline_instance_count` | gauge |  | Number of instances in the ASP.NET application pipeline. |
-| `asp_net_applications.requests_failed` | gauge |  | Count of failed requests in the ASP.NET application |
-| `asp_net_applications.requests_sec` | gauge |  | Rate of requests in the ASP.NET application per second. |
-| `asp_net_applications.session_sql_server_connections_total` | gauge |  | Number of connections to microsoft sql server by an ASP.NET application. |
-| `asp_net_applications.sessions_active` | gauge |  | Number of active sessions in the ASP.NET application. |
-
-
-
+The following table lists the metrics available for this monitor.
+This monitor emits all metrics by default; however, **none are categorized as [container/host/bundled](https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-bundled-and-high-resolution-metrics)**.
```

Comment:
IOW they are all custom? Then say that. And I wouldn't use a different format for the table. Just use the same format and leave Default column blank.

Reply:
The reason we omitted the column in the case of monitors like this is because they actually emit _all_ metrics by default (as the doc says) but _none_ of the metrics are actually _default_/_non-custom_ in our backend, so is it potentially confusing to have the column and have the meaning overloaded like that.  So we just went with having the _Default_ column only when there was a 1:1 correspondence between the agent's default and our categorization so that the _Default_ column has a consistent meaning.

Reply:
Well I just ran across this, which in fact has an empty default column. https://docs.signalfx.com/en/latest/integrations/agent/monitors/internal-metrics.html  I think it is confusing to users to have what looks like the same table format in a bunch of places, but in some cases it's clear whether something is default or not, and in the other it isn't; you have to read the text to understand what the table is showing you. First rule of tech writing: consistency.

Reply:
That is going to be fixed as Mark just recategorized those to be bundled, so they will now show check boxes for all metrics.

Reply:
I don't want to be the word police, but I really suggest folks stop using the word "bundled," since it doesn't match how we document bundled and it could confuse, say, a new person coming aboard and trying to understand how this thing works. I liked the term "included" as the column header, but "default" is also fine. Just sayin' ....